app/assets/stylesheets/submission.css

```diff
@@ -13,6 +13,11 @@ table.results {
   vertical-align: top;
 }
 
+.results .inspect_output th {
```

Comment:
It belongs next to the existing rules for `.results .inspect_test table.inspect_output` further down (the selector should be updated to match).